test/pretty.test.js

```diff
@@ -178,7 +178,7 @@ test('with custom timestamp', async ({ is }) => {
     cb()
```

Comment:
I’m not sure about overloading this test to test two things. I think there should be separate tests to cover both items.